data/blog/tclip-updates-092023.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: tclip updates for September 2023
+date: 2023-09-19
+authors: ['xe']
+tags:
+  - tclip
+  - tsnet
+summary: 'tclip now has a new theme, org-mode rendering, and runs without CGo.'
+images: ['/images/tclip-updates-092023/social.png']
+---
+
+Hey everyone! We've made a bunch of updates to tclip, and we're excited to share them with you. We hope this will help you get the most out of tclip and give you inspiration to make your own addon tools to do whatever it is you want.
+
+[tclip](https://github.com/tailscale-dev/tclip) is a little addon service that gives everyone on your tailnet a place to share small snippets of text, code, log messages, or anything that can fit into a text box. It's like [GitHub Gists](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists) but private on your tailnet.
+
+You can read more about tclip and how to install it on GitHub at [tailscale-dev/tclip](https://github.com/tailscale-dev/tclip). I also wrote more about tclip in [the previous tclip article](/blog/tclip).
+
+This release has a few features and changes that were contributed by the community. Thank you all for helping us!
+
+With huge thanks to:
+
+- [@gmemstr](https://github.com/gmemstr)
+- [@imeyer](https://github.com/imeyer)
+- [@jolheiser](https://github.com/jolheiser)
+
+<HeroImage
+  src="/images/tclip-updates-092023/contrails.jpg"
+  desc="A photo of the East Berlin TV tower with contrails in the sky. Shot on a Nikon D3300 with a 35mm f/1.8 lens by Xe Iaso."
+  width={2000}
+  height={850}
+/>
+
+## New theme
+
+When we initially made tclip, I had hacked up the web UI with my own custom CSS library named [Xess](https://github.com/Xe/Xess). This did work and it helped us get off the ground, but it was not a good long-term solution. It was not very accessible, it was not very pretty, and it was not very easy to maintain.
```

Comment:
This does not bother me, but flagging the brief switch from we to I. I think it's intentional but wanted to note it just in case